tst/com/amazon/corretto/crypto/provider/test/SecurityPropertyTester.java

```diff
@@ -26,11 +29,19 @@ public static void main(String[] args) throws Exception {
     final Provider provider = Security.getProviders()[0];
     assertEquals(NATIVE_PROVIDER.getName(), provider.getName());
 
+    // Ensure that TLS works as expected
+    SSLContext.getInstance("TLS"); // Throws exception on problem
+
     // We know that Java has the SunEC provider which can generate EC keys.
     // We try to grab it to show that the nothing interfered with proper provider loading.
     @SuppressWarnings("unused")
     KeyPairGenerator kpg = KeyPairGenerator.getInstance("EC", "SunEC");
 
+    // Ensure we properly configured ourselves as "strong" instance of SecureRandom
+    // Applications should never use getInstanceStrong as it is an anti-pattern.
+    final SecureRandom strongRng = SecureRandom.getInstanceStrong();
+    assertEquals(NATIVE_PROVIDER.getName(), strongRng.getProvider().getName());
```

Comment:
nit: if we're going to assert that ACCP is the provider here, maybe we should remove the 2 `SUN`  algs from `securerandom.strongAlgorithms`?

Reply:
While we *know* in our tests that ACCP is on a supported system and so the fallbacks aren't needed, we want this configuration to work even when loaded on an unsupported system. Sure, ACCP won't work, but everything else should. So we leave the two SUN algorithms there as fallbacks for that case.